modules/RTC/RTCUtils.js

```diff
@@ -628,14 +628,16 @@ function handleLocalStream(streams, resolution) {
             videoStream = streams.video;
         }
 
-        // Again, different choices on different types of browser.
-        desktopStream = streams.desktopStream || streams.desktop;
+        desktopStream = streams.desktop;
```

Comment:
We were naming the prop `desktopStream` for some browsers and for others `desktop`. I don't believe this was intentional.